config/env.js

```diff
@@ -20,7 +20,9 @@ const {
   STRIPE_PUBLIC_KEY: stripePublicKey,
   SERVICEBOT_ID: servicebotId,
   ALGOLIA_APP_ID: algoliaAppId,
-  ALGOLIA_API_KEY: algoliaAPIKey
+  ALGOLIA_API_KEY: algoliaAPIKey,
+  PAYPAL_CLIENT_ID: paypalClientId,
+  PAYPAL_FIVE_DOLLAR_MONTHLY_PLAN_ID: paypalFiveDollarMonthlyPlanId
```

Comment:
Whats the plan when we add 15 different plans?

Reply:
So the plan id changes between seeding and live that is why we cannot hard code it. since we have a mechanism to distribute the keys and secrets from env I thought it might be a good idea to keep it here.

Please let me know if a better idea comes to mind.

Reply:
I understand it is an identifier and hence it changes. Is it required to be kept a secret? Is it a configuration entry? Can we store this somewhere else? Have we done this already with Stripe's amounts?

Reply:
the sub id is not a secret and is only used when paypal button is clicked to create subscriptions.

I could hard code both ids in config.js and add the logic to write the correct one to config.json during build. how does that sound?

Reply:
Which `config.js`?

Reply:
sorry env.js in /config  https://github.com/freeCodeCamp/freeCodeCamp/blob/master/config/env.js

Reply:
> the sub id is not a secret and is only used when paypal button is clicked to create subscriptions.

Since, its not a secret why can't it be static instead. Did you get a chance to look at: 

https://github.com/freeCodeCamp/freeCodeCamp/blob/master/config/donation-settings.js

Reply:
yes, I did. they could be static but they will change based on environment. so somewhere I will need to check for the production environment and set the right id to be used by the client.